Dalamud/Interface/Internal/Windows/PluginInstaller/PluginChangelogEntry.cs

```diff
@@ -36,7 +35,17 @@ public PluginChangelogEntry(LocalPlugin plugin)
         this.Version = plugin.EffectiveVersion.ToString();
         this.Text = plugin.Manifest.Changelog ?? Loc.Localize("ChangelogNoText", "No changelog for this version.");
         this.Author = plugin.Manifest.Author;
-        this.Date = DateTimeOffset.FromUnixTimeSeconds(this.Plugin.Manifest.LastUpdate).DateTime;
+
+        try
+        {
+            this.Date = DateTimeOffset.FromUnixTimeSeconds(this.Plugin.Manifest.LastUpdate).DateTime;
+        }
+        catch (ArgumentOutOfRangeException)
+        {
+            Log.Verbose("Manifest included improper timestamp, e.g. wrong unit: {PluginName}", plugin.Manifest.Name);
```

Comment:
Please include the exception here at the very least.

I'm unsure how notable this field should be, but this may be worthy of being a warning too?

Reply:
Warning might help to notice it.

I included the exception into the Log call.